src/plugins/intel_cpu/src/nodes/conv.cpp

```diff
@@ -1547,25 +1580,23 @@ void Convolution::initTryBrgconvFlag() {
     // Due to performance issue, brgconv will only be enabled by default:
     // 1, static shape(dynamic shape may change weights layout if the input shape changes and cause performance issue: 86948)
     // 2, support amx
-    // 3, support avx512 except int8 with binary postops
+    // 3, support avx512 without legacy postops/zero point when avx512
     if (!isDynamicNode()) {
         if (dnnl::impl::cpu::x64::mayiuse(dnnl::impl::cpu::x64::avx512_core_amx)) {
             shouldTryBrgconv = true;
         } else if (dnnl::impl::cpu::x64::mayiuse(dnnl::impl::cpu::x64::avx512_core)) {
             shouldTryBrgconv = true;
             // should remove after binary postops performance issue resolved
-            // heuristics: if model has binary post ops we will not use brgconv
-            dnnl::primitive_attr attrs;
-            setPostOps(attrs, MemoryDescUtils::makeDummyShape(getOutputShapeAtPort(0)).getStaticDims(), false);
-            const auto& ops = attrs.get_post_ops();
-            for (int i = 0; i < ops.len(); i++) {
-                if (ops.kind(i) == dnnl::primitive::kind::binary) {
-                    shouldTryBrgconv = false;
-                    break;
-                }
-            }
-            if (shouldTryBrgconv)
-                pInitAttrs[1] = std::make_shared<dnnl::primitive_attr>(std::move(attrs));
+            // heuristics: if it's  avx512 ISA  model && it doesn't have legacy depthwise/quantization post ops or zero point.
+            dnnl::primitive_attr attr;
+            //Pre-set the legacy attr., postop sum data precision need to be updated after preset.
+            setPostOps(attr, MemoryDescUtils::makeDummyShape(getOutputShapeAtPort(0)).getStaticDims(), true);
+            addZeroPoints(attr);
+            attrs.push_back(attr);
+            const auto& ops = attr.get_post_ops();
+            if (!inputZeroPoints.empty() || ops.get()->find(dnnl::impl::primitive_kind::depthwise) != -1 ||
```

Comment:
I can switch back to check binary. What I thought is binary post ops maybe would be dropped and not pushed into attrs vector. But legacy is must needed. Because if post ops has binary-ops, need set legacy. If not having binary post-ops, there should also no legacy post ops, in this case, we can also push attr into vector.